OpenUtau/UI/Strings.xaml

```diff
@@ -47,16 +47,16 @@
     <system:String x:Key="prefs.audio.rendering.resampler.preview">Resampler for preview</system:String>
     <system:String x:Key="prefs.audio.rendering.resampler.export">Resampler for export</system:String>
     <system:String x:Key="prefs.audio.rendering.resampler.note">
-        Note: to use external resamplers, please add resampler DLL or EXE file to Resamplers folder under OpenUtau install path.
+        Note: to use external resamplers, please add resampler DLL or EXE file to the Resamplers folder in the OpenUtau install path.
     </system:String>
     <system:String x:Key="prefs.appearance">Appearance</system:String>
 
     <!-- Dialogs -->
     <system:String x:Key="dialogs.noresampler.caption">No resampler</system:String>
-    <system:String x:Key="dialogs.noresampler.message">No resampler! Put your favourate resampler exe or dll in Resamplers folder!</system:String>
+    <system:String x:Key="dialogs.noresampler.message">No resampler! Put your favourite resampler exe or dll in the Resamplers folder!</system:String>
     <system:String x:Key="dialogs.about.caption">About OpenUTAU</system:String>
     <system:String x:Key="dialogs.about.message">
-        OpenUtau aims to be an open source editing environment for UTAU community, with modern user experience and intelligent phonological support. Visit us on Github.
+        OpenUtau aims to be an open source editing environment for the UTAU community, with a modern user experience and intelligent phonological support. Visit us on Github.
```

Comment:
I believe the "a" is redundant.

Reply:
I don't know how to explain it but it flows better with the a, like, it could be understood without it but I think it is necessary.

Reply:
I think this is a scenario having a standard can be very useful. How about using grammarly.com for grammar check?

Reply:
Whilst I agree a standard would help, I believe you do have to pay to use Grammarly, else I would. I am a native English speaker though if that's anything.

Reply:
Its free version do basic checks and it tells me to remove the 'a'. Fine, you are a native speaker, and I can respect that.